uninstall.php

```diff
@@ -14,5 +14,23 @@
 require_once dirname( __FILE__ ) . '/load-files.php';
 
 $wpephpc = new WPEPHPCompat( dirname( __FILE__ ) );
```

Comment:
I don't see any reason why you're still loading this object here on lines 14 and 16